catalog-info.yaml

```diff
@@ -143,3 +143,47 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: fleet-server-perf-tests
+  description: Pipeline tests for serverless scale testing a PR
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/fleet-server-perf-tests
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: fleet-server-perf-tests
+      description: fleet-server observability-perf tests
+    spec:
+      repository: elastic/fleet-server
+      pipeline_file: .buildkite/pipeline.perf-tests.yaml
+      branch_configuration: "main"
```

Comment:
Should it run also for other branches (7.* or 8.*) ?
NOt sure if this setting is affected when triggering a build using a comment.

Reply:
I don't think we typically run these types of PR scale tests manually on backports